src/SprykerEco/Yves/Payone/Dependency/Client/PayoneToCalculationBridge.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// phpcs:disable SprykerStrict.TypeHints.ReturnTypeHint.MissingNativeTypeHint
```

Comment:
Instead of silencing the error, you should add return type hints. 
Check this RFC: https://spryker.atlassian.net/wiki/spaces/RFC/pages/1023148240/RFC+Return+types

Please, fix this issue in other bridges too.